### DISCUSSION TITLE
Font in post

### DISCUSSION THREAD

**Discussion by evrdev** [mixcore/mix.core#624] [Q&A] [answered]
Hi,  I need to know, how to set  a Font in the post section. Specifically in the content field.

![image](https://user-images.githubusercontent.com/106330995/170514320-1db82fd4-f663-4ed2-81a9-f1bb48a03740.png)

Thank you

**Comment by Smilefounder** [comment, marked as answer]
Hi @evrdev mixcore editor using Trumbowyg editor that you can see the guideline here.
Btw, which mixcore version you are using right now?

https://alex-d.github.io/Trumbowyg/demos/#plugins-fontfamily

**Reply by evrdev** [reply]
Thanks I'll research about that., The mixcore version is 1.0.1.
